agent_verification/Cargo.toml

```diff
@@ -11,13 +11,17 @@ agent_shared = { path = "../agent_shared" }
 anyhow.workspace = true
 async-trait.workspace = true
 chrono.workspace = true
+did-key = "0.2"
 cqrs-es.workspace = true
 futures.workspace = true
 jsonwebtoken.workspace = true
 oid4vc-core.workspace = true
 oid4vc-manager.workspace = true
 oid4vp.workspace = true
-reqwest.workspace = true
+reqwest = { version = "0.12", default-features = false, features = [
+    "json",
+    "rustls-tls",
+] }
```

Comment:
No need to update this dependency (it was changed from:
```toml
reqwest = { version = "0.12", default-features = false, features = [
    "json",
    "rustls-tls",
] }
```

to `reqwest.workspace = true` in a recent commit)